mango/tuner.py

```diff
@@ -36,6 +41,7 @@ class Config:
         exploration_decay: float = 0.9
         exploration_min: float = 0.1
         fixed_domain: bool = False
+        early_stopping: Callable = None
```

Comment:
tiny code optimization to consider... if early_stopping is initialized with `lambda *_: return False` then I think you can remove the need for self.early_stop() etc and simply just call this function directly... 

obviously, totally optional, just maybe simplifies the code... obviously, if you're busy pls ignore!  I'm excited to simply start using this key feature!

Reply:
`lambda` would require call to `deepcopy` even when no callback is provided by the user.